cosmo_tester/test_suites/test_security/auth_test_base.py

```diff
@@ -0,0 +1,550 @@
+########
```

Comment:
There is an essential problem with the way you merged the files. AuthenticationWithoutAuthorization was meant to verify the server's behavior when bootstrapping with an empty authorization section, while the authorization tests verify the server's behavior when an authorization section is part of the blueprint.
The way you merged them loses the first functionality (not using an authorization section at all).
What you want to do here is leave the two test files separately, allow the different bootstrap configurations to be tested, and just extract the assertions. Naming-wise, you should probably name the extracted assertions as "assert_authentication" and "assert_authorization". Furthermore, while the authentication tests use a rest client, authorization tests use cfy. the login method now does both, even though they are never both needed - either the test needs to init a rest client, or it need to set env vars. never both.